db/add_dummy_users.sql

```diff
@@ -1,4 +1,4 @@
-INSERT INTO 
-	users(username, email, password, verified, slackuid)
+INSERT INTO
+	users(username, email, slackuid)
 VALUES
-	('userName', 'user@email.com', 'somepw1', 1, 'userslackid');
\ No newline at end of file
+	('userName', 'user@email.com', 'userslackid');
```

Comment:
This fails with error "Field 'password' doesn't have a default value"

Reply:
You need to do DB reset. Looks like you're using old database. new schema doesn't have password field.

Reply:
Yes - had updated the db but not my .env file...